Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoFirepower.yaml

```diff
@@ -0,0 +1,248 @@
+Normalization:
+  Schema: NetworkSession
+  Version: '0.2.6'
+References:
+  - Title: ASIM Network Session Schema
+    Link: https://aka.ms/ASimNetworkSessionDoc
+  - Title: ASIM
+    Link: https://aka.ms/AboutASIM
+  - Title: Cisco Firepower Documentation
+    Link: 
+      https://github.com/CiscoSecurity/fp-05-firepower-cli/tree/master/estreamer
+      https://www.cisco.com/c/en/us/td/docs/security/firepower/620/api/eStreamer/EventStreamerIntegrationGuide/IS-DCRecords.html#12231
+      https://www.cisco.com/c/en/us/td/docs/security/firepower/620/api/eStreamer/EventStreamerIntegrationGuide/IS-DCRecords.html#88152
+      https://www.cisco.com/c/en/us/td/docs/security/firepower/620/api/eStreamer/EventStreamerIntegrationGuide/RNA-Records.html#48231
+Description: |
+  This ASIM parser supports normalizing commonly used Cisco Firepower messages collected using the CEF Data Connector to the ASIM Network Session normalized schema.
+ParserName: ASimNetworkSessionCiscoFirepower
+EquivalentBuiltInParser: _ASim_NetworkSession_CiscoFirepower
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let ActionLookup = datatable(
+      DeviceAction: string,
+      DvcAction: string,
+      EventResult: string
+  )
+      [
+      "Blocked", "Deny", "Failure",
+      "Alerted", "Allow", "Success",
+      "Rewritten", "Allow", "Success",
+      "Would be Rewritten", "Allow", "Partial",
+      "Would be Blocked", "Deny", "Partial",
+      "Would Be Blocked", "Deny", "Partial",
+      "Dropped", "Drop", "Failure",
+      "Would be Dropped", "Drop", "Partial",
+      "Partially Dropped", "Drop", "Partial",
+      "Would be Block", "Deny", "Partial",
+      "Partial Blocked", "Deny", "Partial",
+      "Rejected", "Deny", "Failure",
+      "Would be Rejected", "Deny", "Partial",
+      "Would Rejected", "Deny", "Partial",
+      "Block", "Deny", "Failure",
+      "Partial Block", "Deny", "Partial",
+      "Drop", "Drop", "Failure",
+      "Would Drop", "Drop", "Partial",
+      "Reject", "Deny", "Failure",
+      "Rewrite", "Allow", "Success",
+      "Allow", "Allow", "Success",
+      "Monitor", "Allow", "Success"
+  ];
+  let EventSeverityLookup = datatable (LogSeverity: string, EventSeverity: string)
+    [
+        "0", "Low",
+        "1", "Low",
+        "2", "Low",
+        "3", "Low",
+        "4", "Medium",
+        "5", "Medium",
+        "6", "Medium",
+        "7", "High",
+        "8", "High",
+        "9", "High",
+        "10", "High"
+    ];
+  let EventResultDetailsLookup = datatable(Reason: string, EventResultDetails: string)
+      [
+      "N/A", "NA",
+      "IP Block", "Failover",
+      "IP Monitor", "Unknown",
+      "User Bypass", "Unknown",
+      "File Monitor", "Unknown",
+      "File Block", "Failover",
+      "Intrusion Monitor", "Unknown",
+      "Intrusion Block", "Unknown",
+      "File Resume Block", "Failover",
+      "File Resume Allow", "Unknown",
+      "File Custom Detection", "Unknown"
```

Comment:
As discussed and suggested [here](https://github.com/Azure/Azure-Sentinel/pull/8412#discussion_r1342572912), we have mapped all Block values as Terminated instead of Failover.